turbopack/crates/turbo-tasks-backend/src/backend/operation/connect_child.rs

```diff
@@ -66,7 +71,7 @@ impl ConnectChildOperation {
             });
         }
 
-        if ctx.should_track_activeness() {
+        if !is_immutable && ctx.should_track_activeness() {
```

Comment:
Can you leave a comment about why we can should activeness tracking for immutable tasks